mobile/context/auth/index.tsx

```diff
@@ -53,17 +48,18 @@ function useProtectedRoute(user: User | null) {
 }
 
 export function Provider(props: PropsWithChildren) {
-  const [user, setAuth] = React.useState<User | null>(null);
+  const account = useAppSelector(selectAccount);
 
-  useProtectedRoute(user);
+  useProtectedRoute(account);
 
   return (
     <AuthContext.Provider
-      value={{
-        signIn: (user) => setAuth(user),
-        signOut: () => setAuth(null),
-        user,
-      }}
+      value={
+        {
+          // signIn: (user) => setAuth(user),
+          // signOut: () => setAuth(null),
```

Comment:
This is commented out. Should it be removed? Or if it should be kept, maybe add a comment to explain?

Reply:
It was removed in the latest commit. Thanks for the notice.